Sources/CoreCommands/SwiftCommandState.swift

```diff
@@ -796,7 +796,7 @@ package final class SwiftCommandState {
 
     private lazy var _toolsBuildParameters: Result<BuildParameters, Swift.Error> = {
         Result(catching: {
-            try _buildParams(toolchain: self.getHostToolchain())
+            try _buildParams(toolchain: self.getHostToolchain(), toolsBuild: true)
```

Comment:
Maybe instead of a boolean flag here we should just pass in the `options.build.buildFlags` or `[]`?

Reply:
Yes, I think that's better and will help when we tidy up the tools/products build option split.